src/test/java/com/plaid/client/integration/RecipientCreateTest.java

```diff
@@ -15,24 +16,68 @@
 public class RecipientCreateTest extends AbstractIntegrationTest {
 
   /**
-   * Utility method that creates a payment recipient.
+   * Utility method that creates a payment recipient with an iban.
    * Used by other integration tests to set up.
    */
-  public static Response<RecipientCreateResponse> createRecipient(PlaidClient client) throws Exception {
+  public static Response<RecipientCreateResponse> createRecipientWithIban(PlaidClient client) throws Exception {
 
     Address address = new Address(Arrays.asList("Street Name 999"), "City", "99999", "GB");
     RecipientCreateRequest recipientCreateRequest =
-      new RecipientCreateRequest("John Doe", "GB33BUKB20201555555555", address);
+        new RecipientCreateRequest("John Doe").withIban("GB33BUKB20201555555555").withAddress(address);
+
 
     Response<RecipientCreateResponse> response =
       client.service().recipientCreate(recipientCreateRequest).execute();
 
     return response;
   }
 
+  /**
+   * Utility method that creates a payment recipient with bacs.
+   * Used by other integration tests to set up.
+   */
+  public static Response<RecipientCreateResponse> createRecipientWithBacs(PlaidClient client) throws Exception {
+
+    Address address = new Address(Arrays.asList("Street Name 999"), "City", "99999", "GB");
+    RecipientCreateRequest recipientCreateRequest =
+        new RecipientCreateRequest("John Doe").withAddress(address).withBacs( new Bacs("12345678", "01-02-03"));
+
+
+    Response<RecipientCreateResponse> response =
+        client.service().recipientCreate(recipientCreateRequest).execute();
+
+    return response;
+  }
+
+  public static Response<RecipientCreateResponse> createRecipientWithBacsAndIban(PlaidClient client) throws Exception {
```

Comment:
wondering if we should add docs somewhere to share that it's valid to create recipients with both IBAN and BACS? or perhaps those docs are already located somewhere (maybe plaid.com)?

Reply:
they are already on the plaid.com docs https://marketing-site-testing-commits.plaid.com/166937/docs#payment-initiation-create-recipient-request